ghost/core/core/shared/config/defaults.json

```diff
@@ -137,7 +137,8 @@
         "version": "2.3"
     },
     "sodoSearch": {
-        "url": "https://unpkg.com/@tryghost/sodo-search@~1.0.0/umd/sodo-search.min.js",
+        "url": "https://unpkg.com/@tryghost/sodo-search@~{version}/umd/sodo-search.min.js",
+        "css": "https://unpkg.com/@tryghost/sodo-search@~{version}/umd/main.css",
         "version": "1.0.0"
```

Comment:
Should we start truncating this to `major.minor` like the Portal one? It'll still resolve to the latest in unpkg and it should also work in jsdelivr because the expanded URL becomes `https://cdn.jsdelivr.net/npm/@tryghost/sodo-search@1.0/umd/sodo-search.min.js`